task3/UI/script_task7.js

```diff
@@ -75,7 +75,7 @@ class AdCollection {
     }
 
     edit(id, adItem) {
-        let mutablePost = this.get(id);
+        let mutablePost = Object.assign(this.get(id));
```

Comment:
If `this.get(id)` will return null, you'll get an error here 😟

Reply:
Thanks!
I keep forgetting about such cases

Reply:
That's OK, the more you practice the less you forget such things 🙂 
Now you can merge the PR.